From a28ce23c20c577587fc1d02eb73a6532cf9eda29 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Wed, 29 Aug 2018 10:44:14 -0700
Subject: [PATCH] MANTA-3125 deliver libmanta via npm registry instead of git
 repo (npmignore file to limit cruft from published package) Reviewed by:
 Jared Morrow <jm@joyent.com> Approved by: Jared Morrow <jm@joyent.com>

---
 .npmignore | 10 ++++++++++
 1 file changed, 10 insertions(+)
 create mode 100644 .npmignore

diff --git a/.npmignore b/.npmignore
new file mode 100644
index 0000000..3136e11
--- /dev/null
+++ b/.npmignore
@@ -0,0 +1,10 @@
+/.gitmodules
+/.dir-locals.el
+/Makefile
+/deps
+/docs
+/test
+/tools
+/tmp
+/npm-debug.log
+/libmanta-*.tgz
-- 
2.21.0

